docs/developer-docs/smart-contracts/maintain/snapshots.mdx

```diff
@@ -19,69 +19,65 @@ Canister snapshots are supported in `dfx` version 0.23.0 and newer.
 
 ## Creating a snapshot
 
-To create a snapshot, first the canister must be stopped. Then, use the `dfx canister snapshot create` command:
+To create a snapshot, first the canister must be stopped. Then, use the [`dfx canister snapshot create <canister-name>`](/docs/current/developer-docs/developer-tools/cli-tools/cli-reference/dfx-canister#dfx-canister-snapshot-create) command:
 
 ```
 dfx canister stop <canister-name>
 dfx canister snapshot create <canister-name>
 ```
 
+<details>
+<summary>Output</summary>
```

Comment:
Should you also make it clear that this is an example of an output or use a generic?

`Created a new snapshot of canister hello_backend. Snapshot ID: <snapshot-id>`